go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/moby/buildkit
 
 go 1.22.0
 
+// FIXME(thaJeztah): testing https://github.com/moby/moby/pull/49162 (pkg_system_volume_uuid branch)
```

Comment:
Added (not yet merged) commit to remove use of pkg/system 🎉 

- https://github.com/moby/moby/pull/49162